scripts/generateConjure.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+set -e
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+function checkCommand() {
+    local DARWIN_COMMAND="$1"
+    local LINUX_COMMAND="$2"
+    local DARWIN_ERROR="$3"
+
+    local COMMAND_NAME="$LINUX_COMMAND"
+    local ERROR_MESSAGE="Aborting"
+
+    if [[ "$OSTYPE" == "darwin"* ]]; then
+        COMMAND_NAME="$DARWIN_COMMAND"
+        ERROR_MESSAGE="$DARWIN_ERROR"
+    fi
+    command -v "$COMMAND_NAME" || { echo >&2 "$COMMAND_NAME is required but not installed. $ERROR_MESSAGE."; exit 1; }
+}
+
+REALPATH=$(checkCommand "grealpath" "realpath" "Try 'brew install coreutils'")
+SPONGE=$(checkCommand "sponge" "sponge" "Try 'brew install moreutils'")
+XQ=$(checkCommand "xq" "xq" "Try 'brew install python-yq'")
+
+
+
+rm -rf "${SCRIPT_DIR}/../tmp"
+
+CONJURE_LITE="npx conjure-lite@0.0.6"
```

Comment:
Does this need to be versioned? We'd need to remember to bump the version here as well as in the `package.json`

Reply:
Oh thanks. yeah we can just use the one.